ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -299,13 +285,7 @@ - (void)logMessage:(NSString*)message level:(ADAL_LOG_LEVEL)level correlationId:
         tokenMessage = [NSString stringWithFormat:@"%@ %@", message, tokenMessage];
     }
     
-    [ADLogger log:level
-          context:self
-          message:tokenMessage
-        errorCode:0
-    correlationId:correlationId
-         userInfo:nil
-           format:@"{\n\tresource = %@\n\tclientId = %@\n\tauthority = %@\n\tuserId = %@\n}",
+    [ADLogger log:level context:self errorCode:0 correlationId:correlationId userInfo:nil isPii:YES format:@"%@ {\n\tresource = %@\n\tclientId = %@\n\tauthority = %@\n\tuserId = %@\n}", tokenMessage,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 199 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)